crates/macros/src/lib.rs

```diff
@@ -54,6 +53,52 @@ use std::path::PathBuf;
 /// ```
 #[proc_macro]
 pub fn import(stream: TokenStream) -> TokenStream {
+    to_tokens(stream)
+}
+
+#[proc_macro]
+pub fn build(stream: TokenStream) -> TokenStream {
+    let tokens = to_tokens(stream);
+
+    // TODO: OUT_DIR is not available from the proc_macro..
+    // let path = PathBuf::from(env::var("OUT_DIR").expect("No `OUT_DIR` env variable set"));
+    let path = PathBuf::from("C:\\git\\rust");
```

Comment:
The `OUT_DIR` environment variable isn't yet available when the proc_macro is executed. What to do...